CHANGELOG.md

```diff
@@ -19,9 +19,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Changed
 
+- [breaking-change] use `fugit::Duraction` for `CountDown::Time` [#381]
+- [breaking-change] Remove all deprecated
```

Comment:
All deprecated what? 🙂

Reply:
Don't pay attention. Accidentally deleted in previous PR